### DISCUSSION TITLE
Uc_driver +pyinstaller issue

### DISCUSSION THREAD

**Discussion by zionkex** [seleniumbase/SeleniumBase#2933] [General]
Hello, Michael.I got a app that use Seleniumbase + PyQt6 and Pyinstaller.When I start app always got that :
Warning: uc_driver not found. Getting it now:

*** chromedriver to download = 126.0.6478.182 (Latest Stable)

Downloading chromedriver-win64.zip from:
https://storage.googleapis.com/chrome-for-testing-public/126.0.6478.182/win64/chromedriver-win64.zip ...
Download Complete!

Extracting ['chromedriver.exe'] from chromedriver-win64.zip ...
Unzip Complete!

The file [uc_driver.exe] was saved to:
C:\Users\PC\AppData\Local\Temp\_MEI78282\seleniumbase\drivers\uc_driver.exe

Making [uc_driver.exe 126.0.6478.182] executable ...
[uc_driver.exe 126.0.6478.182] is now ready for use!
Try to check that directory, but it`s empty. It`s not big deal, but it`s there a way to avoid that problem?If I`m correct, it couldn`t use Driver(path='path/to/driver/directory/')That`s why I`m asking you

**Comment by mdmintz** [comment]
Looks like you're installing SeleniumBase to a temp directory. I would change the install location. The driver location is based on where SeleniumBase is installed, which is the `seleniumbase/drivers/` folder.

**Comment by zionkex** [comment]
Pyinstaller does not create a seleniumbase directory, so seleniumbase will be installed in a temporary directory. I thought the seleniumbase/drivers directory should be installed in the AppData directory. Thanks for the help, now I understand that I just need to create a seleniumbase/drivers/ directory in the pyinstaller folder and the drive will be stored in that directory. You made it very easy, thank you. Can you tell me how you did it? Sorry, I can't find it in the code
